kubetest/main.go

```diff
@@ -459,7 +487,7 @@ func prepare() error {
 		return err
 	}
 
-	if err := os.MkdirAll("./_artifacts", 0777); err != nil { // Create artifacts
+	if err := os.MkdirAll(artifacts, 0777); err != nil { // Create artifacts
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh#L44-L45?

Or if that's not used by e2e test?